specs/common.yaml

```diff
@@ -54,19 +43,19 @@ parameters:
   maxEvents:
     name: maxEvents
     description: >
-        Maximum number of events that server should return at once.
+      Maximum number of events that server should return at once.
     required: false
     in: query
     schema:
       type: integer
       format: int32
       default: 10
-      maximum: 100
       minimum: 1
+      maximum: 100
 
   afterTimestamp:
     name: afterTimestamp
-    description: Apply only to events later than specified timestamp
+    description: Apply only to records created later than the specified timestamp
```

Comment:
maybe good to mention the date-format is `rfc3339`

Reply:
As discussed on TC today - RFC3339 is actually mentioned in https://swagger.io/specification/#data-types